website/app/services/head-data.js

```diff
@@ -1,24 +1,28 @@
 import HeadDataService from 'ember-meta/services/head-data';
+import config from 'ember-get-config';
 
 export default class CustomHeadDataService extends HeadDataService {
-  // this is needed or `head-data.js` will crush
-  get config() {
-    return {};
-  }
-
   get title() {
     return (
       this.currentRouteMeta?.title ??
       this.currentRouteMeta?.frontmatter?.title ??
-      'Helios Design System'
+      config['ember-meta'].title
     );
   }
 
-  get siteName() {
-    return 'Helios Design System';
+  get description() {
+    return (
+      this.currentRouteMeta?.frontmatter?.description ??
+      config['ember-meta'].description
+    );
   }
 
-  get description() {
-    return this.currentRouteMeta?.frontmatter?.description ?? null;
+  get imgSrc() {
+    return this.currentRouteMeta?.id
+      ? `/assets/illustrations/${this.currentRouteMeta?.id.replace(
```

Comment:
not super jazzed about replicating this logic here, but until we have a better solution for these I'm not sure there is another option?

Reply:
If you have capacity, what do you think of opening another PRs that adds support for a `preview-image` property to the frontmatter attributes (as you suggested) and populate it there, so we avoid all of this logic in the code?